Dockerfile

```diff
@@ -81,6 +82,31 @@ COPY --from=bess-build /bin/bessd /bin/bessd
 COPY --from=bess-build /bin/modules /bin/modules
 COPY conf /opt/bess/bessctl/conf
 RUN ln -s /opt/bess/bessctl/bessctl /bin
+
+# Install CNDP dependencies
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    ethtool \
+    libbsd0 \
+    libbpf0 \
+    libelf1 \
+    libgflags2.2 \
+    libjson-c[45] \
+    libnl-3-200 \
+    libnl-cli-3-200 \
+    libnuma1 \
+    libpcap0.8 \
+    pkg-config \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy required CNDP libraries, header files and binaries.
```

Comment:
In line 86, it is already stated that "# Install CNDP dependencies". What about removing these two lines to have everything about CNDP together (no spaces between installation of packages and copy of files [similar to the previous block (Stage bess)])?
Moreover, if you would like, the comment can be changed to "# CNDP: Install dependencies and copy required libraries, header files and binaries" or something like that
```suggestion
```

Reply:
ok